src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -11,6 +11,22 @@ export interface ColumnData {
   sortBy?: SortBy; // optional if the column is not sortable
 }
 
+/**
+ * Data provided for each table cell
+ * Priority of display
+ * 1. customComponent
+ * 2. displayValue
+ * 3. value
+ */
+export interface CustomTableCellData {
```

Comment:
```suggestion
export interface SortableTableCellData {
```

The model should somehow be consistent with the component.

Reply:
Sorry forgot to cleanup from the changes made.